src/tests/Microsoft.PowerFx.Interpreter.Tests/FileExpressionEvaluationTests.cs

```diff
@@ -47,6 +47,19 @@ public void InterpreterTestCase(ExpressionTestCase testCase)
             }
         }
 
+        // Helpful for debugging. Can run a single test. 
```

Comment:
>// Helpful for debugging. Can run a single test.  [](http://example.com/codeflow?start=8&length=49)

We used to have a `RunSingleTestCase` method in this file, which is referenced in the file Testing.txt. Can you either revive this method, or remove Testing.txt?

Reply:
I found it in dataverse tests (different erpo): \PowerApps-Client\src\Language\Tests\PowerFx.Dataverse.Tests\PowerFxEvaluationTests.cs